test/Interop/Cxx/operators/Inputs/member-inline.h

```diff
@@ -6,4 +6,14 @@ struct IntBox {
   IntBox operator+(IntBox rhs) { return IntBox{.value = value + rhs.value}; }
 };
 
+struct HasUserProvidedCopyConstructor {
```

Comment:
Could you rename things a bit?

`IntBox` => `LoadableIntWrapper`
`HasUserProvidedCopyConstructor` => `AddressOnlyIntWrapper`

Reply:
See #33930.